tests/integration/test_logtest/test_remove_old_sessions/test_remove_old_sessions.py

```diff
@@ -22,17 +24,23 @@
 configurations = load_wazuh_configurations(configurations_path, __name__)
 
 # Variables
+logtest_startup_timeout = 30
```

Comment:
Maybe 30 seconds is too much to wait for logtest. Maybe is better to set it to '3'.